documentation/docs/25-build-and-deploy/70-adapter-cloudflare-workers.md

```diff
@@ -11,8 +11,6 @@ To deploy to [Cloudflare Workers](https://workers.cloudflare.com/), use [`adapte
 Install with `npm i -D @sveltejs/adapter-cloudflare-workers`, then add the adapter to your `svelte.config.js`:
 
 ```js
-// @errors: 2307
```

Comment:
You can't just remove these as the site won't build without them. You can test locally by running `pnpm build` in the `kit.svelte.dev` directory